# Definition for SquareNumber.action

# The goal, a single integer to square
int64 number_to_square
---
# The result, the squared number
int64 squared_number
---
# Feedback, will send the intermediate steps of squaring
int64 current_step